src/Microsoft.ML.Data/Training/TrainerUtils.cs

```diff
@@ -382,6 +382,16 @@ public static SchemaShape.Column MakeR4ScalarWeightColumn(string weightColumn)
                 return null;
             return new SchemaShape.Column(weightColumn, SchemaShape.Column.VectorKind.Scalar, NumberType.R4, false);
         }
+
+        /// <summary>
+        /// Check that the label, feature, weights is not supplied in the args of the constructor.
+        /// Those parameters should be internal if they are not used from the maml help code path.
```

Comment:
>maml help code path [](start = 78, length = 19)

Given the focus we now have I'm a little wary of having language specific to the command line in any new code, that is not somehow directly related to the command line.